addons/website/models/website_menu.py

```diff
@@ -152,7 +152,8 @@ def create(self, vals_list):
         return menus
 
     def write(self, vals):
-        self.env.registry.clear_cache('templates')
+        if any(self._ids):
+            self.env.registry.clear_cache('templates')
```

Comment:
why cannot this model use `_clear_cache_name = 'template'`, and have to `clear_cache` before the `super`?
Is it becasue the latter record in `self` has to use the just updated ormcached data of the former record?